app/src/main/java/ch/hikemate/app/model/route/HikeRoutesRepositoryOverpass.kt

```diff
@@ -89,6 +89,28 @@ class HikeRoutesRepositoryOverpass(val client: OkHttpClient) : HikeRoutesReposit
     client.newCall(requestBuilder.build()).enqueue(OverpassSingleRouteHandler(onSuccess, onFailure))
   }
 
+  override fun getRoutesByIds(
+      routeIds: List<String>,
+      onSuccess: (List<HikeRoute>) -> Unit,
+      onFailure: (Exception) -> Unit
+  ) {
+    val overpassRequestDataBuilder = StringBuilder()
+    overpassRequestDataBuilder.append(JSON_OVERPASS_FORMAT_TAG)
+    overpassRequestDataBuilder.append("\n(\n")
+    routeIds.forEach { routeId -> overpassRequestDataBuilder.append("relation($routeId);\n") }
+    overpassRequestDataBuilder.append(");\nout geom;\n")
+
+    val overpassRequestData = overpassRequestDataBuilder.toString()
+
+    val requestBuilder = Request.Builder().url("$OVERPASS_API_URL?data=$overpassRequestData").get()
```

Comment:
Since Overpass request are using `;` as delimiter, new line characters doesn't matter. If you look carefully on the other function implemented, it only trims indents, not new line characters